src/genotet.js

```diff
@@ -9,6 +9,10 @@ $(document).ready(function() {
   genotet.init();
 });
 
+$(window).on('beforeunload', function() {
```

Comment:
It's not a good idea to write global level code. You may add an init function in log such as genotet.logger.init and call that init() in the genotet system initialization (genotet.init).